common/views/components/SectionHeader/SectionHeader.js

```diff
@@ -1,33 +1,42 @@
 // @flow
 import { grid, font } from '../../../utils/classnames';
-import Divider from '../Divider/Divider';
 import Space from '../styled/Space';
+import styled from 'styled-components';
+
+const YellowBox = styled.div`
+  display: inline-block;
+  width: 60px;
+  height: 18px;
+  background: ${props => props.theme.colors.yellow};
+
+  ${props => props.theme.media.medium`
+    width: 58px;
```

Comment:
Not multiples of anything?

Reply:
The width at desktop was what-looked-best, and the scale down to medium/small breakpoints was `what-looked-best / 1.1`. It felt like encoding this would only have added confusion.